docroot/sites/all/themes/ilr_theme/scss/layout/_channel_page.scss

```diff
@@ -58,200 +53,212 @@ body.node-type-channel-page {
+              margin-bottom: 2em;
+              h2, h3 {
+                margin-bottom: .25em;
+              }
+              h4 {
+                font-family: $font;
+                font-size: ms(0);
+                margin-bottom: 1em;
+              }
             }
-            address {
-              font-style: normal;
-              margin-bottom: 1em;
+            .group_unit_information {
+              h2 {
+                @include adjust-font-size-to(22px);
+                font-family: $font;
+                font-weight: 300;
+                line-height: 1;
+                margin-bottom: .25em;
+              }
+              ul.social {
+                margin-bottom: 1em;
+              }
+              address {
+                font-style: normal;
+                margin-bottom: 1em;
+              }
             }
           }
         }
-      }
-
-      .entity-paragraphs-item.paragraphs-item-display-a-block {
-        .event-block {
-          padding-top: 1em;
-          padding-bottom: 1em;
-        }
-        article {
-          @extend %one-column;
-          margin-bottom: 1em !important;
-        }
-        .field-name-field-more-link {
-          text-align: center;
+        &.paragraphs-item-display-a-block {
+          .event-block {
+            padding-top: 1em;
+            padding-bottom: 1em;
+          }
+          article {
+            @extend %one-column;
+            margin-bottom: 1em !important;
+          }
+          .field-name-field-more-link {
+            text-align: center;
+          }
         }
       }
     }
   }
 
   #main[data-eq-state="550"] {
     #content {
-      .entity-paragraphs-item.paragraphs-item-display-a-block {
-        article:not(.node-slide) {
-          @extend %two-column;
-        }
-        article.node-slide {
-          @extend %one-column;
-        }
-      }
-      .entity-paragraphs-item.paragraphs-item-unit-contact-info {
-        .content {
-          .field-name-field-people-list {
-            margin-bottom: 0;
-            @include grid-span (6,1);
+      .entity-paragraphs-item {
+        padding-top: 1em;
+        padding-bottom: 1em;
+        &.paragraphs-item-display-a-block, &.paragraphs-item-compare-three {
+          article:not(.field-name-field-animated-text-card) {
+            @extend %two-column;
+          }
+          article.field-name-field-animated-text-card {
+            @extend %one-column;
           }
-          .group_unit_information {
-            @include grid-span (6,7);
+        }
+        &.paragraphs-item-unit-contact-info {
+          .content {
+            .field-name-field-people-list {
+              margin-bottom: 0;
+              @include grid-span (6,1);
+            }
+            .group_unit_information {
+              @include grid-span (6,7);
+            }
           }
         }
       }
     }
   }
 
-  #main[data-eq-state="768"] {
+  #main[data-eq-state="768"], #main[data-eq-state="700"] {
```

Comment:
If another state is added, how will that affect styles already based on the current eq states? There might be a jumping effect when styles are applied to 550 and 768 but not 700.